io.catenax.binary_exchange/1.0.0/BinaryExchange.ttl

```diff
@@ -0,0 +1,97 @@
+#######################################################################
+# Copyright (c) 2025 Robert Bosch GmbH
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix :<urn:samm:io.catenax.binary_exchange:1.0.0#> .
+
+:BinaryExchange a samm:Aspect ;
+   samm:preferredName "binary exchange"@en ;
+   samm:description "Metadata for exchanging a binary file. Access of the binary file via the dataspace protocol (DSP) is supported."@en ;
+   samm:properties ( :mappingRule :content :binaryFile :dspInformation  ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:mappingRule a samm:Property ;
+   samm:preferredName "mapping rule"@en ;
+   samm:description "Unique identifier of the document specifying the mapping rules used for creating and reading the binary file."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "CX-0123" .
+
+:content a samm:Property ;
```

Comment:
similar to mappingRule: call it "fileContentDescriptionId"
again we might consider to introduce entity from the very beginning to be flexible for extension in the future